### ISSUE TITLE
[Upstream] Download management improvement

### ISSUE THREAD

**Issue by Lordlight0** [fork-maintainers/iceraven-browser#551]
![Screenshot_2023-01-14-14-55-13-55](https://user-images.githubusercontent.com/59312963/212470509-2d9faf02-665e-40b4-b0a0-450ba957f1f7.jpg)
Please fix it !!!

**Comment by klcb2010** [comment]
difficult , you like  mull

**Comment by KaptenLada** [comment]
I feel like a better download manager overall would benefit. Seeing the download in the notifications is not gonna cut it.

**Comment by akliuxingyuan** [comment]
https://bugzilla.mozilla.org/show_bug.cgi?id=1949803

**Comment by coolswapnil** [comment]
Yes, current download manager functionality needs major improvement. Live downloads should be visible in Download tab of browser not just notification panel.
